example-config.yml

```diff
@@ -174,6 +174,16 @@ geocoder:
 
 # Use this mode config for the enhanced Transit+ config
 modes:
+  modeOptions:
+    - mode: TRANSIT
+    - mode: WALK
+    - mode: CAR
+    - mode: BICYCLE
+    - mode: RENT
+      icon: mobile
+      # This overrides the label generated from the i18n. It's not recommended.
+      # label: Rental 
```

Comment:
`label` may probably not be necessary, as the label for the mode can already be overwritten in this file, from the `language` section.

Reply:
That's a good point, I'll go ahead and remove.